src/main/java/net/masterthought/cucumber/reducers/ReducingMethod.java

```diff
@@ -0,0 +1,18 @@
+package net.masterthought.cucumber.reducers;
+
+/**
+ * Supported reducing methods.
+ *
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public enum ReducingMethod {
+
+    /**
+     * Merge features if they have same ID so scenrios will be merged into one scenario.
+     */
+    MERGE_FEATURES_BY_ID,
```

Comment:
Currently I support only one reducing/merging method but I can expect more strategies will be added on request